apps/folder/pkg/apis/folder/v1beta1/folder_spec_gen.go

```diff
@@ -0,0 +1,14 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+
+package v1beta1
+
+// +k8s:openapi-gen=true
+type FolderSpec struct {
+	Title       string  `json:"title"`
+	Description *string `json:"description,omitempty"`
```

Comment:
The big change is that description is now a pointer rather than a string -- description is not really used, so it is OK